scipy/io/wavfile.py

```diff
@@ -742,10 +742,11 @@ def write(filename, rate, data):
     Write to 16-bit PCM, Mono.
 
     >>> from scipy.io.wavfile import write
+    >>> import numpy as np
```

Comment:
This is always assumed on SciPy docs to avoid writing hundreds of identical lines, hence needs to be omitted

Reply:
Thanks ilayn, sorry for my ignorance.

Reply:
No problem at all. Thank you for your time.